src/DSP/CombFilter.cpp

```diff
@@ -16,7 +16,7 @@ CombFilter::CombFilter(Allocator *alloc, unsigned char Ftype, float Ffreq, float
     unsigned int srate, int bufsize)
     :Filter(srate, bufsize), memory(*alloc), sr(srate), gain(1.0f), type(Ftype)
 {
-    mem_size = sr/30;
+    mem_size = sr/24;
```

Comment:
Seems like this should be buffer_size+maximum_delay_lookback if I'm reading the rest of the code correctly. I think with the current value it is assuming something like buffer_size caps out at around 128 with 48kHz sample rate. (Not a terrible place to put it, but give users the choice and they'll break the assumption).

Reply:
> Seems like this should be buffer_size+maximum_delay_lookback if I'm reading the rest of the code correctly. I think with the current value it is assuming something like buffer_size caps out at around 128 with 48kHz sample rate. (Not a terrible place to put it, but give users the choice and they'll break the assumption).

the buffer gets accessed via
(mem_size-buffersize+i)-delayfwdbuf[i])
this index has to be > 0 for linear sampling and >1 for 3rd order sampling
while
delayfwd = sr/ff
(mem_size-buffersize+i)-sr/ff > 1
(mem_size-buffersize+i) > sr/ff + 1
mem_size >  sr/ff + 1 + buffersize -i
minimal ff: 25
so i would say:
mem_size = sr/24 + buffersize
is safe under all circumstances

Reply:
```
(mem_size-buffersize+i)-delayfwdbuf[i] must be >= 0
minimum value with i=0
delayfwdbuf is maximal at samplerate/25.0
so mem_size-buffersize-samplerate/25.0 >=0
mem_size >= samplerate/25.0+buffersize
```

sr/24+buffersize also works, though I'm likely missing something with the 25 vs 24 difference.

Reply:
> mem_size >= samplerate/25.0+buffersize
> ```
> 
> sr/24+buffersize also works, though I'm likely missing something with the 25 vs 24 difference.

it was just a safety margin due to truncation of the division result sr/24 and a few samples more for higher order interpolation. but maybe adding a few samples is more straight forward.

Reply:
Fair enough for me.